hardware/optical_power_meters.py

```diff
@@ -18,7 +18,27 @@
 """
 
 import visa
+from hardware import u
 
+class MockOpticalPowerMeter:
+
+     def __init__(self, instr_name = None):
+        if not instr_name:
+            self.name = "Optical Power Meter - Newport 1830C"
+        else:
+            self.name = instr_name
+        self._power = 5 * u.watt
+
+    def identify(self):
+        return self.name
+
+    @property
+    def power(self):
+        return self._power
+
+    @power.setter
+    def power(self, value):

```

Comment:
We'll probably want to double check the units here. But this device doesn't get used that frequently. Put it in the issue tracker